proposals/NNNN-Addition-of-a-way-to-notify-HMI-of-the-required-support-for-apps.md

```diff
@@ -0,0 +1,103 @@
+# Addition of the way to notify HMI of the required support for apps
+
+* Proposal: [SDL-NNNN](NNNN-Addition-of-a-way-to-notify-HMI-of-the-required-support-for-apps.md)
+* Author: [Akihiro Miyazaki (Nexty)](https://github.com/Akihiro-Miyazaki)
+* Status: **Awaiting review**
+* Impacted Platforms: [ Core / Java Suite / HMI / RPC / Protocol ]
```

Comment:
I do not see any RPC Spec (MOBILE_API) changes listed within this proposal, only HMI_API changes, which is included in the Core platform. Please add the MOBILE_API changes or remove "RPC" from the `Impacted Platforms` and `Impact on existing code` sections.

If RPC Spec changes are required, note that iOS and JavaScript Suite platforms will also be impacted (see https://github.com/smartdevicelink/sdl_evolution#rpc-spec), and this proposal should be updated accordingly.

Reply:
We will remove RPC Spec from the `Impacted Platforms` and `Impact on existing code`.